src/libp2p.js

```diff
@@ -97,6 +100,9 @@ const createLibp2p = async ({
   listenAddrs.forEach(addr => {
     peerInfo.multiaddrs.add(multiaddr(addr))
   })
+  const connEncryption = []
+  if (secio !== false) connEncryption.push(SECIO)
+  if (noise) connEncryption.push(new Noise(null, null, true))
```

Comment:
```suggestion
  if (noise) connEncryption.push(NOISE)
```
This works also, because we removed third param in rc.8